common/views/components/EventPromo/EventPromo.js

```diff
@@ -89,9 +126,19 @@ const EventPromo = ({
             {event.title}
           </Space>
 
-          {event.isOnline && !event.availableOnline && (
-            <OnlineIndicator>Online</OnlineIndicator>
-          )}
+          {event.isOnline && !event.availableOnline && <OnlineIndicator>Online</OnlineIndicator>}
```

Comment:
So if it is available online but already happened, we don't show the play icon?

Reply:
Should the logic be `isOnline && isPast`?

Reply:
I think this is right as is - it just shows it is an online event